src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -659,6 +659,13 @@ export default TreeBuilderToolbar
   }
 }
 
+.viamo-app-container .tree-builder-toolbar {
+  position: sticky;
```

Comment:
This is awesome, I didn't have that in mind before, but sticking the toolbar with this calc width is genius.
💯 

Approving!